build.js

```diff
@@ -0,0 +1,77 @@
+"use strict";
+
+const fs = require('fs');
+const https = require('follow-redirects').https;
+const archiver = require('archiver');
+
+let options = {
+    host: 'uplift.shipit.staging.mozilla-releng.net',
+    path: '/coverage/supported_extensions',
+    method: 'GET',
+    headers: {"Content-Type": "text/plain"}
+}
+
+
+let req = https.get(options, (res) => {
```

Comment:
You can do directly: `https.get('https://uplift.shipit.staging.mozilla-releng.net/coverage/supported_extensions', res => `

Reply:
Yes but its a long line

Reply:
It's OK, just 99 characters.